shared/templates/static/oval/sshd_expected.xml

```diff
@@ -0,0 +1,72 @@
+<def-group>
```

Comment:
@redhatrises Thank you, I have separated the OVAL definitions and moved some logic into them. SSH OVALs are much more simpler now.